### PR TITLE
Updated ReadMe and added missing package

### PR DESCRIPTION
**Pull request by EverettRhodes**
- Updated Readme with latest setup steps
- Added an example settings.json file for the function app
- added a missing package for the React app

### CODE REVIEW

**Review by rsimosa — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,16 +5,47 @@
...
+   - **Note: No need to run DbUp if you imported bacpac files for the DBs** 
+   - **Need two databases WaDE2 and WaDE2Test**
+      - WaDE2Test is for the unit tests
+   - If not using the default SQL server, update the connection string in launchSettings.json in the DbUp project
+   - Example: `Data Source=localhost;Initial Catalog=WaDE2;User Id=sa;Password=password`
    - May need to run this twice if using docker
-5. Start Function App
-6. Start React App
+5. Setup Function App
+   1. Right-click on **Client.Functions** project and choose **Set as Startup Project**
```
Comment (goldilocked) [inline README.md:20]: Please make these into - instead of 1s

File: README.md [hunk 2]
```diff
@@ -5,16 +5,47 @@
...
+   1. Set the listening port to 5001
+      1. Right-click on **Client.Functions** project
+      1. Choose Properties
+      1. Select Debug -> General -> Open debug launch profiles UI
+      1. In the command line arguments enter the following
+         1. `host start --port 5001`
+      1. Run the Function App
+6. Setup React App
+   - install NPM packages.
+      - From the DashboarUI directory `npm install`
```
Comment (goldilocked) [inline README.md:32]: dashboardUI is mispelled

**Review by goldilocked — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (EverettRhodes) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (EverettRhodes) [reply README.md]: fixed

**Pull request merged**